### ISSUE TITLE
Business intake form for OAR/VERA Proactive Schedule Texts

### ISSUE THREAD

**Issue by christophercolvin** [department-of-veterans-affairs/va.gov-team#67866]
## Your Details
1.  Shaun Chelgreen
​
2.  **Business Line**
    VBA Office of Transition Economic Development and Office of Administrative Review - 
​
## Notification Details
1.  **Notification type**
    Email campaign (bulk messaging) to acquire Opt-In for SMS messaging
​
2.  **Please describe your use case.**
    OAR seeks to proactively send out text messages to eligible Veterans to encourage them to set up appointments for Higher Level Reviews (HLR) in the Veteran Engagement and Reporting Application (VERA).  However, before text messages can be sent out proactively, an email campaign would need to be completed first to solicit potential Veterans to opt-in to SMS messaging. 
​
3.  **What actions can the user take based on the notification they receive?**
   For the Email message - Veterans can choose to opt-in to receiving text messages
  For the SMS message - Veterans can choose to set up an appointment in VERA
​
4.  **What is the desired business outcome?**
    Increase number of HLR appointments which would create efficiencies in the appeals process.
​
5.  **What system will kick off the notification? Please note the system should be inside the VA or have an ATO.**
    May be a manual data transfer or automated from VBMS/VACOLS data pertaining to eligible Veterans.
​
6.  **What will trigger the notification?**
    The upload of the data whether manual or automated.  The data is based on a list of eligible Veterans for HLRs.
​
7.  **Provide sample content per notification type, if you have it.**
   We do not have this at this time.
​
8.  **Has a Privacy Officer (PO) seen and approved the content? If not, do you know who your PO is?**
    PO has not seen or approved any of this content.  Unknown at this time which PO is assigned to OAR.
​
9.  **Would you prefer to provide contact information for the recipients or would you rather VA Notify look this up for you by Veteran ID? If by ID, please let us know what identifier is used in your system.**
   Preference is for VA Notify to look up Veteran ID via MPI based on most likely BCID
​
10.  **Do you currently capture communication preferences related to this notification? If so, please describe.**
    We do not believe we currently capture communication preference at this time in our systems.
​
11.  **What is the anticipated volume of notifications per day, week, month?**
    Potentially 1000s per month
​
13.  **When does this notification need to be in production?**
    TBD but most likely ASAP per OAR.

**Comment by christophercolvin** [comment]
POC is Shaun Chelgreen and Christopher Colvin
Shaun.Chelgreen1@va.gov
Christopher.Colvin@va.gov

**Comment by mjones-oddball** [comment]
Eran reached out to me via teams with some questions. Posting here with my responses.

> is VA Notify able to send bulk emails out? i.e. is it whitelisted so that we don't get our IP flagged if we need to send a message to several thousand veterans? For vera we are looking at a need to send an email to veterans asking them to opt in to text messages before we bulk send them a text with a link to VERA for setting up an appointment.  I'm also wondering if there is a way through VA Notify to get a list of veterans who have already opted in to receiving SMS from the VA?

Answers below!

There are notification preferences in VA Profile, which are surfaced on VA.gov and other sites. Since there isn't an existing preference for VERA, that's something we'd work on submitting an intake for with VA Profile. "I'm also wondering if there is a way through VA Notify to get a list of veterans who have already opted in to receiving SMS from the VA?" -- there isn't just one preference for SMS as it's based on the type of message and service being offered. For example, there is a preference for appointment reminders for medical appointments.

VA Notify doesn't offer a service to tell you who all is opted in, but you could ask VA Profile what options are available via integration with them depending on what you're trying to do there.

If VA Notify is looking up the contact info and opt in/out from VA Profile (once a preference is available) then all you need to do is send me the SMS request for those who are eligible to schedule appts and have not already done so. I believe we can use a default send strategy for this use case which means if the Veteran has provided their mobile phone number and not explicitly opted out to these notifications, then VA Notify can send the text.

As for the SMS blasts - our API supports individual requests and there are throughput considerations to account for, so we can discuss more about that as we work through all the use case details and steps.

Here's what the preferences look like on VA.gov. They are making some design updates and adding email channel soon so it may look slightly different in the near future, but same concept. 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/99204969/7e4b6431-24fe-4a87-9795-c5232af6269f)
